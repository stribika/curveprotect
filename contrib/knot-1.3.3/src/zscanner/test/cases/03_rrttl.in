$ORIGIN	.
$TTL	1

; OK
@	0		NS	@	; Minimal ttl
@	3600		NS	@	; Inner value
@	4294967295	NS	@	; Maximal ttl
@	1S		NS	@	; 1 second (upper-case)
@	1s		NS	@	; 1 second (lower-case)
@	1M		NS	@	; 1 minute (upper-case)
@	1m		NS	@	; 1 minute (lower-case)
@	1H		NS	@	; 1 hour (upper-case)
@	1h		NS	@	; 1 hour (lower-case)
@	1D		NS	@	; 1 day (upper-case)
@	1d		NS	@	; 1 day (lower-case)
@	1W		NS	@	; 1 week (upper-case)
@	1w		NS	@	; 1 week (lower-case)
@	1w1d1h1m1s	NS	@	; More time units
@	1s1m1m		NS	@	; Same time units, non decreasing order

; KO
@	-1		NS	@	; Negative ttl
@	4294967296	NS	@	; 32bit overflow
@	100000000W	NS	@	; 32bit overflow
@	4294967295s1w	NS	@	; 32bit overflow
@	1x		NS	@	; Unknown time unit
