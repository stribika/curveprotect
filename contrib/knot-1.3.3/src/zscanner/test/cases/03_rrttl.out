OWNER=00
CLASS=0001
RRTTL=00000000
RTYPE=0002
RDATA=00
------
OWNER=00
CLASS=0001
RRTTL=00000E10
RTYPE=0002
RDATA=00
------
OWNER=00
CLASS=0001
RRTTL=FFFFFFFF
RTYPE=0002
RDATA=00
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0002
RDATA=00
------
OWNER=00
CLASS=0001
RRTTL=00000001
RTYPE=0002
RDATA=00
------
OWNER=00
CLASS=0001
RRTTL=0000003C
RTYPE=0002
RDATA=00
------
OWNER=00
CLASS=0001
RRTTL=0000003C
RTYPE=0002
RDATA=00
------
OWNER=00
CLASS=0001
RRTTL=00000E10
RTYPE=0002
RDATA=00
------
OWNER=00
CLASS=0001
RRTTL=00000E10
RTYPE=0002
RDATA=00
------
OWNER=00
CLASS=0001
RRTTL=00015180
RTYPE=0002
RDATA=00
------
OWNER=00
CLASS=0001
RRTTL=00015180
RTYPE=0002
RDATA=00
------
OWNER=00
CLASS=0001
RRTTL=00093A80
RTYPE=0002
RDATA=00
------
OWNER=00
CLASS=0001
RRTTL=00093A80
RTYPE=0002
RDATA=00
------
OWNER=00
CLASS=0001
RRTTL=000A9A4D
RTYPE=0002
RDATA=00
------
OWNER=00
CLASS=0001
RRTTL=00000079
RTYPE=0002
RDATA=00
------
WARNG=ZSCANNER_EBAD_NUMBER
------
WARNG=ZSCANNER_ENUMBER32_OVERFLOW
------
WARNG=ZSCANNER_ENUMBER32_OVERFLOW
------
WARNG=ZSCANNER_ENUMBER32_OVERFLOW
------
WARNG=ZSCANNER_EBAD_TIME_UNIT
------
